erpnext/www/support/index.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import unicode_literals
+import frappe, json
+from frappe import _
+from frappe.utils.user import is_website_user
+import requests
+
+def get_context(context):
+	context.no_cache = 1
+	context.align_greeting = ''
+	context.align_search_box = 'search-input-alignment-left'
+	s = frappe.get_doc("Support Settings")
+
+	context.greeting_text = s.greeting_text if s.greeting_text else _("We're here to help")
+
+	if s.greeting_text_and_search_bar_alignment == 'Center':
+		context.align_greeting = 'text-center'
+		context.align_search_box = 'search-input-alignment-center'
+	if s.greeting_text_and_search_bar_alignment == 'Right':
+		context.align_greeting = 'text-end'
+		context.align_search_box = 'search-input-alignment-right'
```

Comment:
Don't set css classes here, set them in jinja. 
Manage search and text alignment using single css class